.storybook/stories/dropdown/dropdown-item-that-opens-modal.stories.ts

```diff
@@ -4,50 +4,64 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { ClrDropdownModule, ClrModalModule } from '@clr/angular';
-import { Parameters } from '@storybook/addons';
-import { Story } from '@storybook/angular';
+import { ClrDropdown, ClrDropdownModule, ClrModalModule } from '@clr/angular';
+import { moduleMetadata, Story, StoryObj } from '@storybook/angular';
+import { CommonModules } from 'helpers/common';
 
-import { setupStorybook } from '../../helpers/setup-storybook.helpers';
+export default {
+  title: 'Dropdown/Dropdown With Modal',
+  decorators: [
+    moduleMetadata({
+      imports: [...CommonModules, ClrDropdownModule, ClrModalModule],
+    }),
+  ],
+  component: ClrDropdown,
+  argTypes: {
+    clrCloseMenuOnItemClick: { defaultValue: true, control: { type: 'boolean' } },
+  },
+  args: {
+    clrCloseMenuOnItemClick: true,
+  },
+};
 
-const defaultStory: Story = args => ({
+const DropdownModalTemplate: Story = args => ({
   template: `
-    <clr-dropdown [clrCloseMenuOnItemClick]="clrCloseMenuOnItemClick">
-      <button class="btn btn-outline-primary" clrDropdownTrigger>
-        Dropdown
-        <cds-icon shape="angle" direction="down"></cds-icon>
-      </button>
-      <clr-dropdown-menu>
-        <div clrDropdownItem (click)="modalOpen = true">Open Modal</div>
-        <div clrDropdownItem>Do Nothing</div>
-        <clr-dropdown>
-          <button clrDropdownTrigger>Nested Trigger</button>
-          <clr-dropdown-menu>
-            <div clrDropdownItem (click)="modalOpen = true">Open Modal</div>
-            <div clrDropdownItem>Do Nothing</div>
-          </clr-dropdown-menu>
-        </clr-dropdown>
-      </clr-dropdown-menu>
-    </clr-dropdown>
+    <div style="margin-bottom:500px; text-align: center">
```

Comment:
We don't need that line and it's breaking the modal.

Reply:
Removed the center alignment to fix the breaking modal, kept the bottom margin to 100px so that the dropdown menu is visible in the Docs.